application/F3DOptionsTools.cxx

```diff
@@ -192,14 +192,19 @@ void PrintHelpPair(
   std::string_view key, std::string_view help, int keyWidth = 10, int helpWidth = 70)
 {
   std::stringstream ss;
-  ss << "  " << std::left << std::setw(keyWidth) << key << " " << std::setw(helpWidth) << help;
+  ss << "  " << std::left << std::setw(keyWidth) << key;
+  if (key.size() > static_cast<size_t>(keyWidth))
+  {
+    ss << "\n  ";
```

Comment:
@mwestphal do we want the `help` string to still be aligned? if so:
```suggestion
    ss << "\n  " << std::setw(keyWidth) << " ";
```
as per PR:
```
Examples:
  f3d file.vtu -xtgans                               View a unstructured mesh in a typical nice looking sciviz style       
  f3d file.glb -tuqap --hdri-file=file.hdr --hdri-ambient --hdri-skybox
   View a gltf file in a realistic environment                           
  f3d file.ply -so --point-size=0 --comp=-2          View a point cloud file with direct scalars rendering                 
  f3d folder                                         View all files in folder   
```
with suggestion:
```
Examples:
  f3d file.vtu -xtgans                               View a unstructured mesh in a typical nice looking sciviz style       
  f3d file.glb -tuqap --hdri-file=file.hdr --hdri-ambient --hdri-skybox
                                                     View a gltf file in a realistic environment                           
  f3d file.ply -so --point-size=0 --comp=-2          View a point cloud file with direct scalars rendering                 
  f3d folder                                         View all files in folder 
```

Reply:
indeed, you are right @snoyer